packages/back-end/src/models/FeatureRevisionModel.ts

```diff
@@ -90,10 +100,12 @@ export async function getRevisions(
     .limit(25);
```

Comment:
when moving to another env do we want to remove the limit so that all revisions are passed over? or we might want to add something in the UI that will state only the previous 25 will be moved